api/desecapi/mail_backends.py

```diff
@@ -34,7 +35,9 @@ def _run_task(messages, debug, **kwargs):
         logger.warning('Sending queued email, details: %s', debug)
         kwargs.setdefault('backend', kwargs.pop('backbackend', MultiLaneEmailBackend.default_backend))
         with get_connection(**kwargs) as connection:
-            return connection.send_messages([dict_to_email(message) for message in messages])
+            content = connection.send_messages([dict_to_email(message) for message in messages])
+            metrics.get('desecapi_sent_messages_count').inc()
```

Comment:
actually, it would be nice to have a histogram here, and use `num_sent` as the value!

Reply:
need to discuss before implementing